engine/render/renderer/platform/vulkan/Texture2D.cpp

```diff
@@ -72,7 +72,7 @@ void Texture2D::LoadFromFile(const String& filePath)
 
     uint64_t imageSize = texWidth * texHeight * 4;
 
-    uint8_t pixelPtr[imageSize];
+    uint8_t* pixelPtr = new uint8_t[imageSize];
```

Comment:
Why?

Reply:
The Windows stack size is very small and trying to allocate this size statically was causing crashes on Windows. This was necessary to prevent any major crashes on Windows from loading textures.